src/components/graph/nodes/network-modification-node.js

```diff
@@ -119,9 +116,13 @@ const NetworkModificationNode = (props) => {
                                 style={{ margin: 'auto' }}
                             />
                         )) ||
-                            (props.data.buildStatus === 'NOT_BUILT' && (
-                                <VisibilityOffIcon style={{ margin: 'auto' }} />
-                            ))}
+                            (props.data.buildStatus === 'NOT_BUILT' &&
+                                //Setup for a different display mode, replace false by a configuration variable
+                                false && (
```

Comment:
I think you can remove it if this is not used for the moment.
It is planned in short term to implement another display mode ?

Reply:
Ok I just had the answer in the backlog refinement :) So you can keep it yes